### PR TITLE
docs(arrow-integration): Arrow Integration on Malibu Advanced

### PR DESCRIPTION
**Pull request by nandap4790**
# Description

This PR handles the documentation for the Arrow integration on Malibu advanced.

Preview: https://github.com/quintype/malibu/blob/arrow-integration/tutorial/arrow-integration-malibu-advanced.md

https://github.com/quintype/ace-planning/issues/466

### CODE REVIEW

**Review by ags1773 — APPROVED** [review]

File: tutorial/arrow-integration-malibu-advanced.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+title: Arrow Integration to Malibu Advanced
+parent: Malibu Advanced Tutorial
```
Comment (Deokumarjnu) [inline tutorial/arrow-integration-malibu-advanced.md:3]: Did `Malibu Advanced Tutorial` exist as parent ? i think you need to create a file and assign it as parent , see how `Malibu Tutorial` has been written as parent .

Comment (nandap4790) [reply tutorial/arrow-integration-malibu-advanced.md:3]: sure will check it

**Review by Deokumarjnu — APPROVED** [review]
LGTM

File: tutorial/arrow-integration-malibu-advanced.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+Apart from building custom components on the Malibu Advanced app, a wide range of components that are built as a part of the [Arrow](https://developers.quintype.com/quintype-node-arrow/) can also be included and customized based on the requirements.
+
+The list of the Arrow components can be found [here](https://developers.quintype.com/quintype-node-arrow/).
+
+As you clone the Malibu Advanced repository, you will be getting some of the Arrow components out of the box.
+
+Ex: ElevenStories, FourColTwelveStories, etc.
+
+**Steps to add a new Arrow component to your app:**
```
Comment (Athira001) [inline tutorial/arrow-integration-malibu-advanced.md:18]: Currently, this feature is available in Malibu-Advanced. Could you please mention that if they cloned before  DD/MM  they need to follow the below steps to integrate the arrow?

**Review by Athira001 — APPROVED** [review]

**Pull request merged**